src/components/Card/CardReadMore.js

```diff
@@ -3,30 +3,38 @@ import PropTypes from 'prop-types'
 import classNames from 'classnames'
 
 import Icon from '../Icon/Icon'
+import tagPropType from 'reactstrap/src/utils'
 
 const propTypes = {
   className: PropTypes.string,
   iconName: PropTypes.string,
   href: PropTypes.string,
-  text: PropTypes.string
+  text: PropTypes.string,
+  tag: tagPropType,
+  to: PropTypes.string
 }
 
 const defaultTypes = {
   iconName: 'it-arrow-right'
 }
 
+const defaultProps = {
```

Comment:
Can you put here also the `iconName` prop?
It was a typo to name it that way, it had to be `defaultProps`.